WordPress/Classes/ViewRelated/Menus/MenuItemsViewController.m

```diff
@@ -9,6 +9,10 @@
 #import "ContextManager.h"
 #import "Menu+ViewDesign.h"
 #import <WordPressShared/WPDeviceIdentification.h>
+#import "WPGUIConstants.h"
+
+static CGFloat const ItemHoriztonalDragDetectionWidthRation = 0.5;
```

Comment:
Fixed in https://github.com/wordpress-mobile/WordPress-iOS/commit/562ea8a0521cf9d92a7e360283fa440d07e7db79